iothub/device/src/DeviceAuthenticationWithTokenRefresh.cs

```diff
@@ -63,5 +57,19 @@ public override IotHubConnectionStringBuilder Populate(IotHubConnectionStringBui
             iotHubConnectionStringBuilder.DeviceId = DeviceId;
             return iotHubConnectionStringBuilder;
         }
+
+        private static int SetSasTokenSuggestedTimeToLiveSeconds(int suggestedTimeToLiveSeconds)
+        {
+            return suggestedTimeToLiveSeconds == 0
```

Comment:
int can be negative

Reply:
The validation is a part of the base class: https://github.com/Azure/azure-iot-sdk-csharp/blob/master/iothub/device/src/AuthenticationWithTokenRefresh.cs#L36-L44

Reply:
Better not to assume, tho. Simply make it <= to be safe.